test/dotnet.Tests/ParserTests/MigrateParserTests.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Tools.Migrate;
+using Xunit;
+using Xunit.Abstractions;
+using Parser = Microsoft.DotNet.Cli.Parser;
+
+namespace Microsoft.DotNet.Tests.ParserTests
+{
+    public class MigrateParserTests
+    {
+        public MigrateParserTests(ITestOutputHelper output)
+        {
+            this.output = output;
+        }
+
+        private readonly ITestOutputHelper output;
+
+        [Fact]
+        public void MigrateParserConstructMigrateCommandWithoutError()
```

Comment:
That's not what I meant, I meant we should a similar set of command line arguments and make sure they are parsed properly. It should still be a parser unit test.